gm4_balloon_animals/beet.yaml

```diff
@@ -0,0 +1,25 @@
+id: gm4_balloon_animals
+name: Balloon Animals
+version: 1.0.X
+
+data_pack:
+  load: .
+
+pipeline:
+  - gm4_balloon_animals.generate_loot_tables
+  - gm4.plugins.extend.module
+  - gm4.plugins.include.lib_trades
+
+meta:
+  gm4:
+    versioning:
+      required:
+        lib_trades: 1.3.0
+      schedule_loops: [main]
+    website:
+      description: Looking for exotic animals? This module will add a Wandering Trader that sells animals!
```

Comment:
I think saying 'This module will add a Wandering Trader that [...]' makes it sound like the player has to do something special to get such a trader. Maybe write '[...] makes some Wandering Traders sell animals!' instead?